core/domain/prod_validators_test.py

```diff
@@ -15364,7 +15364,9 @@ def setUp(self):
         self.user_id = self.get_user_id_from_email(USER_EMAIL)
 
         wipeout_service.pre_delete_user(self.user_id)
-        wipeout_service.delete_user(
+        wipeout_service.run_user_deletion(
```

Comment:
Could you explain this change?

If it's not obvious, maybe leave a comment in the code.

Reply:
Done. Added comment at the top.